### DISCUSSION TITLE
Улучшения UI/UX в зачетной книжке

### DISCUSSION THREAD

**Discussion by Dragonprod** [0niel/university-app#206] [Идеи]
**Проблема:**
В текущей вкладке с зачётной книжкой информация представлена в виде таблицы с некоторой ненужной информацией для студента(Дата, ФИО преподавателя). 

**Решение:**
Оценки за экзамены/зачеты должны быть представлены в виде карточки, которую можно развернуть и увидеть подробности. 

**Основная информация на карточке:**

- Название предмета
- Вид экзамена(зачет/экзамен и тд)
- Оценка

**Подробности на карточке:**

- Дата проведения экзамена
- ФИО преподавателя

**Comment by Dragonprod** [comment]
**TODO:**
- [x] Создать макет

**Comment by ValeryVerkhoturov** [comment]
| Название предмета | Оценка |   |
|-------------------|--------|---|
| Иностранный язык  | Зачет  | + |
| Мат анализ        | 4      | + |
| Программирование  | 5      | - |
* Экзамен
* 10.01.2022
* Иванов Иван Иванович

**Comment by Dragonprod** [comment]
Реализовано
